graylog2-server/src/main/java/org/graylog2/streams/StreamServiceImpl.java

```diff
@@ -95,6 +95,7 @@ public Stream create(CreateStreamRequest cr, String userId) {
         streamData.put(StreamImpl.FIELD_CREATOR_USER_ID, userId);
         streamData.put(StreamImpl.FIELD_CREATED_AT, Tools.iso8601());
         streamData.put(StreamImpl.FIELD_CONTENT_PACK, cr.contentPack());
+        streamData.put(StreamImpl.FIELD_MATCHING_TYPE, cr.matchingType().toString());
```

Comment:
In this case the `@Nullable` on the getter should be removed as only the method argument in the factory method is nullable but not the actual attribute: https://github.com/Graylog2/graylog2-server/pull/1292/files#diff-1480db0af34470adda8a855ab7063cb5R49